manifests/view.pp

```diff
@@ -9,6 +9,7 @@
     $recursion_match_clients      = 'any',
     $recursion_match_destinations = '',
     $recursion_match_only         = false,
+    $empty_zones                  = '',
```

Comment:
Sure, but we can't set a default Boolean value if we want to remain compatible with Bind's defaults across versions (as mentioned in my commit message).

Relevant excerpts:

> Although the default setting is 'yes', users of BIND 9.6-ESV-R5, 9.7.4, 9.8.1 (and superseding revisions and minor versions) will not see the new RFC 1918 empty zones in addition to the existing automatic empty zones unless they explicitly configure empty-zones-enable yes;

> For a view that has recursion enabled, the default is "empty-zones-enable yes;"

> For a view that has recursion disabled, the default is "empty-zones-enable no;"